src/test/Calculator.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Good job including snapshot tests for the Calculator component, however you need to add tests simulating user interactions using the React Testing Library. Kindly add them.

  You can visit these links for idea on how to simulate user actions:
  -  [https://testing-library.com/docs/react-testing-library/intro/](https://testing-library.com/docs/react-testing-library/intro/)
  - [https://www.robinwieruch.de/react-testing-library/](https://www.robinwieruch.de/react-testing-library/)

Reply:
Thanks for the links

Reply:
Fixed.